jsettlers.graphics/src/main/java/jsettlers/graphics/map/minimap/Minimap.java

```diff
@@ -159,49 +159,47 @@ private void drawViewmark(GLDrawContext context) {
 		if (mapViewport == null) {
 			return;
 		}
-		int lineStartX = mapViewport.getLineStartX(0);
-		int firstY = mapViewport.getLineY(0);
-		float minviewx = converter.getViewX(lineStartX, firstY, 0) * width;
-		float maxviewy = Math.min(converter.getViewY(lineStartX, firstY, 0), 1) * height;
-		float maxviewx = converter.getViewX(mapViewport.getLineEndX(0), firstY, 0)
-				* width;
-		int lastY = mapViewport.getLineY(mapViewport.getLines());
-		float minviewy = Math.max(converter.getViewY(lineStartX, lastY, 0), 0) * height;
+		int offset = 50;
```

Comment:
Why do you exactly need 50?

Reply:
It was a trial and error. The result is a good fit. I did not find the underlying problem.

Reply:
MapContent uses a screen padding of 50, maybe this plays together?

Reply:
Yes, this should be it. We should move it to a common constant so that if we change it one day, we won't miss the minimap.

Reply:
Not sure if MapContent#SCREEN_PADDING = 50; is already the right constant and should be public.

Reply:
Could you try to find out if its the right constant by chaning it locally to see what it changes and if it changes the minimap screen size accordingly?

Reply:
I don't see a difference when increasing the value of SCREEN_PADDING or OVERDRAW_BOTTOM_PX. So I don't think there is a correlation.

Reply:
Then leave it as a separate constant.